src/test/encoding/check-generated.sh

```diff
@@ -17,23 +19,35 @@ for type in `./ceph-dencoder list_types`; do
     num=`./ceph-dencoder type $type count_tests`
     echo "$num $type"
     for n in `seq 1 1 $num 2>/dev/null`; do
-	if ! ./ceph-dencoder type $type select_test $n encode decode; then
+	safe_type=$type
+	# BitVector<2> needs some escaping to avoid bash issues with <>
+	if [ "$type" = "BitVector<2>" ]; then
+	    safe_type="BitVector\<2\>"
+	fi
+
+	pids=""
+	run_in_background pids bash -c "./ceph-dencoder type $safe_type select_test $n dump_json > $tmp1"
```

Comment:
I'm not sure because of the redirection. I don't really check how it would be handled.